<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceRecordPage</class>
 <widget class="QWidget" name="PlaceRecordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_place">
     <property name="text">
      <string>Введите место</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_place">
     <property name="placeholderText">
      <string>Место</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_countPlace">
     <property name="text">
      <string>Введите количесво мест</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_placeCount">
     <property name="placeholderText">
      <string>Количество мест</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_add">
     <property name="text">
      <string>Дополнительные условия</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_cms_5">
     <property name="text">
      <string>Минимум 5 участников с КМС</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_ms_5">
     <property name="text">
      <string>Минимум 5 участников с МС</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
